src/khoj/routers/helpers.py

```diff
@@ -1477,10 +1478,16 @@ def get_user_config(user: KhojUser, request: Request, is_detailed: bool = False)
 
     user_subscription_state = get_user_subscription_state(user.email)
     user_subscription = adapters.get_user_subscription(user.email)
+
     subscription_renewal_date = (
         user_subscription.renewal_date.strftime("%d %b %Y")
         if user_subscription and user_subscription.renewal_date
-        else (user_subscription.created_at + timedelta(days=7)).strftime("%d %b %Y")
+        else None
+    )
+    subscription_enabled_trial_at = (
+        user_subscription.enabled_trial_at.strftime("%d %b %Y")
+        if user_subscription and user_subscription.enabled_trial_at
+        else None
```

Comment:
Should we set `subscription_enabled_trial_at` to `user_subscription.created_at + timedelta(days=7)).strftime("%d %b %Y")` if `enabled_trial_at` is unset for some reason? The downstream dependencies (e.g common/auth.ts, settings/page.tsx on web app) seem to expect it to be a (non-nullable) string

Reply:
`subscription_enabled_trial_at` is important to keep state of whether the trial has been used or not, so I don't think we should fill in an arbitrary value like that. I'll make the type nullable on the TypeScript object instead.